satpy/readers/viirs_vgac_l1c_nc.py

```diff
@@ -75,9 +75,7 @@ def set_time_attrs(self, data):
 
     def dt64_to_datetime(self, dt64):
         """Conversion of numpy.datetime64 to datetime objects."""
-        if isinstance(dt64, np.datetime64):
-            return dt64.astype(dt.datetime)
-        return dt64
+        return datetime64_to_pydatetime(dt64)
```

Comment:
Is it worth it to keep this method? Or can we just replace it with the new utility function?

Reply:
I was hesitating because it's a public method, but we can probably replace it. It's only used once in the tests.